commit 7373a31771637f68d372334ba0deadfa5e49b3ff (refs/changes/36/36/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-07-06T14:57:37-07:00 (3 years, 3 months ago)
    
    TOOLS-1455: sdc-scripts.git:sbin/postlogrotate.sh can attempt to cat log files over themselves

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 9cd96a3..9a78955 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 9cd96a35d84c223485e14f02e7a4ebca2088ca51
+Subproject commit 9a78955bf23c8b4d3c0dd083c69f062a456f33a5
